modules/data.atmosphere/R/tdm_generate_subdaily_models.R

```diff
@@ -0,0 +1,226 @@
+##' Create statistical models to predict subdaily meteorology from daily means
+##' @family tdm - Temporally Downscale Meteorology
+##' @title gen_subdaily_models
+##' @param outfolder - directory where models will be stored *** storage required varies by size of training dataset, but prepare for >100 GB
+##' @param dat.train_file - train_data file
+##' @param in.prefix 
+##' @param tdf_file - tdm_temporal_downscale_functions.R filepath that can be sourced i.e. '~/scripts/temporal_downscale_functions.R'
+##' @param n.beta - number of betas to save from linear regression model
+##' @param resids - logical stating whether to pass on residual data or not
+##' @param parallel - logical stating whether to run temporal_downscale_functions.R in parallel 
+##' @param n.cores - deals with parallelization
+##' @param day.window - integer specifying number of days around a particular day you want to use data from for that 
```

Comment:
We need our times to match up at the same time of day as the previous one to generate the statistics, so it has to be in a daily interval so we can pull stats from 13:00 on both days for example